test/e2e_node/pkg/remote/remote.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package e2e_node
+package remote
```

Comment:
Adding the extra `pkg` in the path seems a bit unnecessary here. What makes this one go in `pkg` while other stuff is in `runner`? Maybe this should be `test/e2e_node/runner/remote`?